Customer.DomainServices/Services/CustomerService.cs

```diff
@@ -1,58 +1,75 @@
 ﻿using Customer.DomainModels.Formatters;
 using Customer.DomainModels.Models;
 using Customer.DomainServices.Services.Interfaces;
+using Customer.Infrastructure.Data.Context;
+using Microsoft.EntityFrameworkCore;
 
 namespace Customer.DomainServices.Services
 {
     public class CustomerService : ICustomerService
     {
-        private readonly List<CustomersModel> customersList = new();
+        private readonly FeatureContext _featureContext;
+        private readonly DbSet<CustomerModel> _customers;
 
-        public List<CustomersModel> Get()
+        public CustomerService(FeatureContext featureContext)
         {
-            return customersList;
+            _featureContext = featureContext ?? throw new ArgumentNullException(nameof(featureContext)); 
+            _customers = _featureContext.Set<CustomerModel>();
         }
 
-        public long Create(CustomersModel customer)
+        public IList<CustomerModel> Get()
         {
-            if (customersList.Any(x => x.Email == customer.Email || x.Cpf == customer.Cpf))
+            return _customers.ToList();
```

Comment:
Esse ToList está aqui apenas para consumir mais memória e degradar performance
```suggestion
            return _customers;
```